src/schema/v2/Alerts/index.ts

```diff
@@ -177,157 +177,190 @@ export const AlertType = new GraphQLObjectType<
-    },
-    additionalGeneNames: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ additional_gene_names }) => additional_gene_names,
-    },
-    artistIDs: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ artist_ids }) => artist_ids,
-    },
-    artists: {
-      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(ArtistType))),
-      resolve: async ({ artist_ids }, _args, { artistsLoader }) => {
-        if (!artist_ids) return []
+  fields: () => {
+    const {
+      filterArtworksConnectionWithParams,
+    } = require("../filterArtworksConnection")
 
-        const { body } = await artistsLoader({ ids: artist_ids })
-        return body ?? []
+    return {
+      ...IDFields,
+      acquireable: {
+        type: GraphQLBoolean,
       },
-    },
-    artistsConnection: {
-      type: new GraphQLNonNull(artistConnection.connectionType),
-      args: pageable(),
-      resolve: async ({ artist_ids }, args, { artistsLoader }) => {
-        if (!artist_ids || artist_ids.length === 0) return emptyConnection
+      additionalGeneIDs: {
+        type: new GraphQLList(GraphQLString),
+        resolve: ({ additional_gene_ids }) => additional_gene_ids,
+      },
+      additionalGeneNames: {
+        type: new GraphQLList(GraphQLString),
+        resolve: ({ additional_gene_names }) => additional_gene_names,
+      },
+      artistIDs: {
+        type: new GraphQLList(GraphQLString),
+        resolve: ({ artist_ids }) => artist_ids,
+      },
+      artists: {
+        type: new GraphQLNonNull(
+          new GraphQLList(new GraphQLNonNull(ArtistType))
+        ),
+        resolve: async ({ artist_ids }, _args, { artistsLoader }) => {
+          if (!artist_ids) return []
+
+          const { body } = await artistsLoader({ ids: artist_ids })
+          return body ?? []
+        },
+      },
+      artistsConnection: {
+        type: new GraphQLNonNull(artistConnection.connectionType),
+        args: pageable(),
+        resolve: async ({ artist_ids }, args, { artistsLoader }) => {
+          if (!artist_ids || artist_ids.length === 0) return emptyConnection
 
-        const { page, size } = convertConnectionArgsToGravityArgs(args)
-        const { body } = await artistsLoader({ ids: artist_ids })
+          const { page, size } = convertConnectionArgsToGravityArgs(args)
+          const { body } = await artistsLoader({ ids: artist_ids })
 
-        const totalCount = body.length
-        return {
-          totalCount,
-          pageCursors: createPageCursors({ page, size }, totalCount),
-          ...connectionFromArray(body, args),
+          const totalCount = body.length
+          return {
+            totalCount,
+            pageCursors: createPageCursors({ page, size }, totalCount),
+            ...connectionFromArray(body, args),
+          }
+        },
+      },
+      artworksConnection: filterArtworksConnectionWithParams((args) => {
+        const filterArtworksArgs = {
```

Comment:
similar to what you @anandaroop mentioned [here](https://github.com/artsy/metaphysics/pull/5573#discussion_r1496267356) - another case of duplication, but this time in Gravity format. Not sure if I should extract it to a common place so it could be re-used later 🤷